From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Wed, 6 Mar 2019 19:54:49 +0100
Subject: icontheme: Fix annotations on array arguments

The extra ':' makes gir not to handle properly all the rest.

Origin: upstream, 2.24.33, commit:072d8546b07f2fd6d41cfbf891e4efeab6dca0cc
---
 gtk/gtkicontheme.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gtk/gtkicontheme.c b/gtk/gtkicontheme.c
index 60661c8..ea40460 100644
--- a/gtk/gtkicontheme.c
+++ b/gtk/gtkicontheme.c
@@ -747,7 +747,7 @@ gtk_icon_theme_set_search_path (GtkIconTheme *icon_theme,
 /**
  * gtk_icon_theme_get_search_path:
  * @icon_theme: a #GtkIconTheme
- * @path: (allow-none): (array length=n_elements) (out): location to store a list of icon theme path directories or %NULL
+ * @path: (allow-none) (array length=n_elements) (out): location to store a list of icon theme path directories or %NULL
  *        The stored value should be freed with g_strfreev().
  * @n_elements: location to store number of elements
  *              in @path, or %NULL
@@ -3219,7 +3219,7 @@ gtk_icon_info_get_embedded_rect (GtkIconInfo  *icon_info,
 /**
  * gtk_icon_info_get_attach_points:
  * @icon_info: a #GtkIconInfo
- * @points: (allow-none): (array length=n_points) (out): location to store pointer to an array of points, or %NULL
+ * @points: (allow-none) (array length=n_points) (out): location to store pointer to an array of points, or %NULL
  *          free the array of points with g_free().
  * @n_points: (allow-none): location to store the number of points in @points, or %NULL
  * 
